airbyte-integrations/connector-templates/source-python/secrets/config.json.hbs

```diff
@@ -1,3 +1,3 @@
 {
-  "fix-me": "TODO populate with needed configuration for integration tests or delete this file and any references to it. The schema of this file should match what is in your spec.yaml"
+  "TODO": "TODO populate with needed configuration for integration tests or delete this file and any references to it. The schema of this file should match what is in the spec section of your manifest.yaml"
```

Comment:
This doesn't make really make for Python sources. They don't have a manifest to define a spec and they still require an external `spec.yaml` file to define the expected configs